app/utils/sweats_message_handler.js

```diff
@@ -0,0 +1,141 @@
+import { DiscordRequest } from './discord_request.js';
+import { EventEmitter } from 'events';
+const green=0x66FF33
+const red=0xff6633
```

Comment:
(Suggestion: architecture) I think these should be abstracted to a file called `colors.js` in `/utils/` and imported here instead